<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>manageGrocery</class>
 <widget class="QWidget" name="manageGrocery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="groceryTableWidget">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>11</y>
     <width>901</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>209</y>
     <width>901</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Quantity</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>309</y>
     <width>901</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTextEdit" name="nameEdit"/>
    </item>
    <item>
     <widget class="QTextEdit" name="quantityEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="addGroceryButton">
      <property name="text">
       <string>Add grocery</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
